public/test/mocks/react-inlinesvg.tsx

```diff
@@ -1,6 +1,17 @@
-import React from 'react';
+import React, { Ref } from 'react';
 
-export default function ReactInlineSVG({ src, innerRef, cacheRequests, preProcessor, ...rest }) {
+export default function ReactInlineSVG({
```

Comment:
Due to [including the test utils](https://github.com/grafana/grafana/pull/85672/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R22) and aliasing, the tsconfig has pulled in this file (it happened to be the only `tsx` file in the test folder). It was complaining about `any` typing so I added the bare minimum here